src/OpenTelemetry.Instrumentation.Owin/Implementation/DiagnosticsMiddleware.cs

```diff
@@ -20,6 +20,8 @@ internal sealed class DiagnosticsMiddleware : OwinMiddleware
     private static readonly Func<IOwinRequest, string, IEnumerable<string>> OwinRequestHeaderValuesGetter
         = (request, name) => request.Headers.GetValues(name);
 
+    private static readonly RequestDataHelper RequestDataHelper = new(configureByHttpKnownMethodsEnvironmentalVariable: false);
```

Comment:
Consider follow up PR and setting it to true. Extended documentation and changelog entry needed.